native/src/routes/PoisContainer.tsx

```diff
@@ -21,6 +21,8 @@ const PoisContainer = ({ navigation, route }: PoisContainerProps): ReactElement
   const { slug, multipoi } = route.params
   const { cityCode, languageCode } = useCityAppContext()
 
+  const [enteredViaMaps] = useState(slug === undefined)
```

Comment:
I think you don't have to pass a state here?
You have the `slug` also in `Pois`